app/home/home.html

```diff
@@ -2,7 +2,7 @@
     <div class="ibox-content">
         <div class="row home">
             <div class="col-xs-4 col-sm-3">
-                <img class="img-circle img-responsive center-block m-b-sm" src="static/images/demo/profile_photo.jpg" alt="profile photo">
+                <img class="img-circle img-responsive center-block m-b-sm" src="static/images/demo/profile_photo.jpg" alt="profile photo" ng-click="vm.UploadUserLogo()">
```

Comment:
Please, wrap this image in empty link (`<a hre=""></a>`), so user can undestend, that this image is clickable.
And set max height and max width for image.